packages/components/src/__future__/Tag/AvatarTag.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Avatar, AvatarProps } from "@kaizen/draft-avatar"
+import { BaseTag } from "./BaseTag"
+import { Dismiss, DismissProps } from "./Dismiss"
+
+export type AvatarTagProps = AvatarProps & {
+  children: string
+  onDismiss?: DismissProps["onDismiss"]
+}
+
+export const AvatarTag = ({
```

Comment:
Please remove this component as we haven't finished designing for all the possible colour combinations with the Avatar's no image state, so we can't technically release with this.